tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/FmiIntegrationTests.cs

```diff
@@ -75,15 +76,14 @@ public async Task Flow1_RmaCredential_From_CertTestAsync()
             //Assert
             AssertResults(authResult,
                           confidentialApp,
-                          "-login.windows.net-atext-4df2cbbb-8612-49c1-87c8-f334d6d065ad-f645ad92-e38d-4d1a-b510-d1b09a74a8ca-api://azurefmitokenexchange/.default-zm2n0e62zwtsnnsozptlsooob_c7i-gfpxhyqqinjuw",
+                          "-login.microsoftonline.com-atext-4df2cbbb-8612-49c1-87c8-f334d6d065ad-f645ad92-e38d-4d1a-b510-d1b09a74a8ca-api://azurefmitokenexchange/.default-zm2n0e62zwtsnnsozptlsooob_c7i-gfpxhyqqinjuw",
```

Comment:
can all of this go into a helper method and return `$"-login.microsoftonline.com-atext-{clientId}-{tenantId}-{scope}-{fmiHash}".ToLowerInvariant();`